File: testdata/bench/swift/BenchmarkTest00010.swift
import Vapor
import SQLite3

func handler010(_ req: Request) throws -> String {
    let sortBy: String = try req.query.get(at: "sort")
    let direction = req.query[String.self, at: "dir"] ?? "ASC"
    let query = "SELECT * FROM products ORDER BY \(sortBy) \(direction)"
    var db: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_exec(db, query, nil, nil, nil)
    return "sorted"
}

File: testdata/bench/cpp/BenchmarkTest00010.cpp
#include <string>
#include <sqlite3.h>

void search_like(const std::string &pattern) {
    std::string query = "SELECT * FROM items WHERE name LIKE '%" + pattern + "%'";
    sqlite3 *db;
    sqlite3_open("app.db", &db);
    sqlite3_exec(db, query.c_str(), nullptr, nullptr, nullptr);
    sqlite3_close(db);
}
